docs/src/main/asciidoc/security-oidc-auth0-tutorial.adoc

```diff
@@ -0,0 +1,587 @@
+[id="security-oidc-auth0-tutorial"]
+= Protect Quarkus web application by using Auth0 OpenID Connect provider
+include::_attributes.adoc[]
+:diataxis-type: tutorial
+:categories: security,web
+
+xref:security-architecture.adoc[Quarkus Security] provides comprehensive OpenId Connect (OIDC) and OAuth2 support with its `quarkus-oidc` extension, supporting both xref:security-oidc-code-flow-authentication.adoc[Authorization code flow] and xref:security-oidc-bearer-token-authentication.adoc[Bearer token] authentication mechanisms.
```

Comment:
```suggestion
xref:security-architecture.adoc[Quarkus Security] provides comprehensive OpenID Connect (OIDC) and OAuth2 support  through the `quarkus-oidc` extension, which supports both xref:security-oidc-code-flow-authentication.adoc[Authorization Code Flow] and xref:security-oidc-bearer-token-authentication.adoc[Bearer token] authentication mechanisms.
```

Reply:
I'm not sure this one works. I'd like to start with 1 sentence overview of what Quarkus OIDC can do